src/wp-includes/kses.php

```diff
@@ -2395,6 +2395,14 @@ function safecss_filter_attr( $css, $deprecated = '' ) {
 		'list-style-image',
 	);
 
+	/*
+	 * CSS attributes that accept rgba color data types.
+	 *
+	 */
+	$css_rgba_color_data_types = array(
+		'background-color',
+	);
```

Comment:
```suggestion
	$css_color_data_types = array(
		'color',
		'border-color',
		/* ETC ETC for all of the items that support color values in the safe CSS list */
		'background-color',
	);
```

A generic name to allow for other color types to be added later: there are currently specs been written for new color types so it will become a thing sooner rather than later.